tck/features/expressions/pattern/Pattern1.feature

```diff
@@ -30,4 +30,293 @@
 
 Feature: Pattern1 - Existential Pattern Match
 
+  Scenario: [1] Matching on any single outgoing directed connection
+    Given an empty graph
+    And having executed:
+      """
+      CREATE (a:A)-[:REL1]->(b:B), (b)-[:REL2]->(a), (a)-[:REL3]->(:C), (a)-[:REL1]->(:D)
+      """
+    When executing query:
+      """
+      MATCH (n) WHERE (n)-->() RETURN n
```

Comment:
```suggestion
      MATCH (n) WHERE exists( (n)-->() ) RETURN n
```

Reply:
I have no confirmation yet, however:

 * the CIP does not mention `exists( <pattern> )`
 * `size( <pattern> )` gets deprecated because if this kind of ugly syntax ambiguity between parenthesized expressions and node patterns
 * `exists( <pattern> )` suffers the same kind of ugly syntax ambiguity between parenthesized expressions and node patterns

Hence, it would be kind of odd to keep `exists( <pattern> )`. So, based on that thinking, it looks much more likely that is deprecated too and hence it seems smarter to not invest into `exists( <pattern> )` any effort. 

Further, assuming `exists( <pattern> )` goes out like `size( <pattern> )` does, the category `pattern` will on contain pattern comprehensions. That give us the possibility to spilt pattern comprehension into multiple feature. However, that can happen in separate PR at a different time. Just something to keep in mind.

Reply:
I just realized that `size(<pattern>)` fails on the openCypher grammar, when the `<pattern>` is a node pattern. I reckon `exists(<pattern>)` will fail, too. From viewpoint of the grammar both are simply [`FunctionInvocation`s](https://github.com/opencypher/openCypher/blob/master/grammar/basic-grammar.xml#L381-L387), which only allow expressions as parameters. [Expressions](https://github.com/opencypher/openCypher/blob/master/grammar/basic-grammar.xml#L296-L317) currently allow [`RelationshipsPattern`s](https://github.com/opencypher/openCypher/blob/master/grammar/basic-grammar.xml#L360-L363), but I assume that will go away, too. 

Anyway, the practical consequence of this is that scenarios that test the failing of these functions on a pattern argument require a `@skipGrammarCheck` tag, otherwise the integrity check will complain: 
```the query of step `When executing query:` in line XX has either a syntax conforming to the grammar or the scenario has the @skipGrammarCheck tag```

Reply:
Update: Given the state of the discussion on `<pattern>` and `exist( <pattern> )` in expressions that I am observing, the best is probably to not touch `pattern/Pattern1.feature` either was for the time being. 

Nevertheless, working on the existential subquery part in a separate `existentialSubquery` category is good. That is stable.